platforms/core-runtime/build-profile/src/integTest/groovy/org/gradle/profile/ConfigurationCacheBuildProfileIntegrationTest.groovy

```diff
@@ -36,7 +37,8 @@ class ConfigurationCacheBuildProfileIntegrationTest extends AbstractConfiguratio
         file("src/main/java/included/Example.java") << ""
 
         when:
-        configurationCacheRun(":help", "--profile")
+        executer.withConfigurationCacheEnabled()
```

Comment:
💭 

Could override `setupExecuter()` to perform this configuration automatically